react/components/TelemarketingContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import { compose, path } from 'ramda'
```

Comment:
This change is just moving code from the root of the react dir to the components dir